doc/develop/toolchains/zephyr_sdk.rst

```diff
@@ -20,6 +20,12 @@ The Zephyr SDK supports the following target architectures:
 * x86 (32-bit and 64-bit)
 * Xtensa
 
+By default the Zephyr build system assumes the toolchain in use to be the Zephyr
```

Comment:
I think this paragraph is redundant and should be removed. Either that or

- rework the existing mentions of these variables below
- just say "By default, the Zephyr build system assumes that the toolchain will be obtained from the Zephyr SDK."